extmod/webrepl/websocket_helper.py

```diff
@@ -65,10 +72,23 @@ def client_handshake(sock):
 Sec-WebSocket-Key: foo\r
 \r
 """)
-    l = cl.readline()
-#    print(l)
-    while 1:
+    if not ssl:
```

Comment:
Dumb question: why does the handshake have to be different for wss?

Reply:
The main reason of a big percentage of changes is due to sockets and ssl sockets  not having the same methods. SSL socket doesn't have a `.makefile` neither `send` or `recv` as a normal socket.